user_install.sh

```diff
@@ -60,10 +60,10 @@ MILRC_FILE=$MIL_CONFIG_DIR/milrc
 #======================#
 
 # Install no project by default, the user must select one
-SELECTED=false
-INSTALL_SUB=false
-INSTALL_PRO=false
-INSTALL_NAV=false
+SELECTED="false"
```

Comment:
See #3 for the answer to this. It mainly boils down to doing installs on SubjuGator and Shuttle, where the main script has already been run. The script generates, populates, and builds a user workspace without needing any root access. It was how I made all of your workspaces for the last pool day.